portfolio/src/main/webapp/script.js

```diff
@@ -28,12 +28,34 @@ const hideMenuShowMain = () => {
   }
 }
 
- /**
-  * ADD EVENT LISTENERS
-  */
+/* HOME */
+const actions = document.getElementsByClassName('action');
+// indicates the action which is currently shown
+let currAction = 0;
+// animates actions by displaying them like a slideshow
+const actionsAnimate = () => {
+  const nextAction = (currAction + 1) % actions.length;
+  // make current invisible
+  actions[currAction].style.display = 'none';
+  // show next
+  actions[nextAction].style.display = 'inline';
+  currAction = nextAction;
+}
+
+/**
```

Comment:
Please structure your comments, it doesn't quite make sense have three blocks of comments stacked together.

Reply:
I see... I guess I will remove the last two then. 😅

Reply:
Please resolve this conversation if you have made the change

Reply:
I don't see the comment being resolved, there are still three blocks of comments here

Reply:
Hi! Sorry about the confusion. It has been resolved in another branch (week1-nav-interactive). I haven't rebased that branch here. Hence, it looks like it has not been updated. I will update them later this evening.

Reply:
Sure no rush, feel free to resolve the conversation when you have done so.

Reply:
Done! Thanks for the patience :D